lib/galaxy/tools/toolbox/panel.py

```diff
@@ -112,7 +112,7 @@ def to_dict(self, **kwds):
         return super(ToolSectionLabel, self).to_dict()
 
 
-class ToolPanelElements(odict, HasPanelItems):
+class ToolPanelElements(OrderedDict, HasPanelItems):
```

Comment:
We discussed this on Gitter, these were the main points:
- The `insert()` method was added to `odict` in commit https://github.com/galaxyproject/galaxy/commit/7853e71dad4cfbde60a847197586769b91d2def3 .
- There is no alternative implementation that solves the tool panel use case and is as efficient as `odict`. Especially given that Python 3.6 has `OrderedDict` implemented in C, so we cannot add an `insert` method to it any more.
- If we end up keeping `odict` for this use case, it may make sense to leave it where it is since it has been distributed as part of `galaxy-lib` and now `galaxy-util` packages.
- We can remove the unused `reverse()` method of `odict`.